tools/build_standard_commutations.py

```diff
@@ -95,6 +104,19 @@ def _get_unparameterizable_gates() -> List[Gate]:
     return [g for g in gates if len(g.params) == 0]
 
 
+@lru_cache(None)
```

Comment:
I'm a bit confused about the caching, as I thought that would be useful if we call the function many times -- though it appears we're only calling it once. But this also applies to the function above 🤔 @brandhsn could you maybe help us here? 🙂

Reply:
That is a good point. I only added it because I was copypasting the function above. I will let @brandhsn answer, but if there is no good reason to keep it, I can remove it.

Reply:
You can remove it, thanks.

Reply:
Thanks @sbrandhsn! 🙂